src/utils/mask-data.ts

```diff
@@ -0,0 +1,55 @@
+import { Injectable } from '@nestjs/common';
+import { createCipheriv, createDecipheriv } from 'crypto';
+
+
+const secretKey = Buffer.from(process.env.SECRET_KEY, 'base64');
+
+export function encrypt(data: string): string {
+    const cipher = createCipheriv('aes-256-ecb', secretKey, null);
+    const encrypted = Buffer.concat([cipher.update(data, 'utf8'), cipher.final()]).toString('base64');
+    return encrypted;
+  }
+
+  export function decrypt(encryptedData: string): string {
+    const decipher = createDecipheriv('aes-256-ecb', secretKey, null);
+    const decrypted = Buffer.concat([decipher.update(Buffer.from(encryptedData, 'base64')), decipher.final()]).toString('utf8');
+    return decrypted;
+  }
+
+
+export function maskMobileNumber(mobileNumber: any): string {
```

Comment:
@souravbhowmik1999 Is any specific reason for using any?
mobileNumber: any Cant we use string?